packages/core/src/__tests__/useRoute.test.tsx

```diff
@@ -37,3 +37,34 @@ test('gets route prop from context', () => {
     </BaseNavigationContainer>
   );
 });
+
+test('gets parent route from context', () => {
```

Comment:
We should add some more tests for `useRoute(name)`:

- returns route for the current route
- returns route for the parent route
- throws error for invalid route
- throws error for child route